### PR TITLE
Added translation to brazilian portuguese

### PR DESCRIPTION
**Pull request by rafaelpivatto**
Hello guys.

This pull request contains all text translated to brazilian portuguese (pt).

Regards.

### CODE REVIEW

**Comment by msarilar** [comment]
hey, great job!

the PR conflicts with the master branch because other people updated the russian and german translation. I don't think the conflict will be hard to solve ; if you manage to do it and update the PR then great, otherwise I'll try and take care of it next weekend

**Comment by wesleymoura** [comment]
Thanks for the warning, I'll tell the others and in the evening here in Brazil when everyone is home already, let's see if today or at most tomorrow we have already settled the conflicts and update the PR :)

**Comment by rafaelpivatto** [comment]
That's right now, conflicts are solved.
Tks.

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -34,2845 +39,3251 @@
...
       "es": "Mostrar",
+      "pt": "Mostrar",
       "de": "Anzeigen",
       "fr": "Afficher",
       "ru": "Открыть",
       "pl": "Pokaż"
     },
     "Toggle Window Mode (Locked/Unlocked)": {
       "es": "Cambiar el modo ventana (Bloqueado/Desbloqueado)",
+      "pt"
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: missing comma at the end of the line

**Comment by msarilar** [comment]
hey thanks a lot, that was fast :)

one small mistake and I'll merge this in master ; expect a deployment very soon after it

**Comment by rpivatto** [comment]
Good sounds @msarilar 
Thank you very much.

And one more thing... hahahaha
Many thanks to translation collaborators: CMDRs Moraes, Wladimir JRC and Warlord from brazilian minor faction Cobra Wing. :tada:

**Comment by msarilar** [comment]
thanks a lot!

**Pull request merged**